exporter/skywalkingexporter/logdata_to_logrecord.go

```diff
@@ -0,0 +1,163 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package skywalkingexporter
+
+import (
+	"go.opentelemetry.io/collector/model/pdata"
+	conventions "go.opentelemetry.io/collector/translator/conventions/v1.5.0"
+	common "skywalking.apache.org/repo/goapi/collect/common/v3"
+	logpb "skywalking.apache.org/repo/goapi/collect/logging/v3"
+	"strconv"
+	"time"
+)
+
+const (
+	spanIDField            = "spanID"
+	severityNumber         = "severityNumber"
+	severityText           = "severityText"
+	name                   = "name"
+	flags                  = "flags"
+	instrumentationName    = "otlp.name"
+	instrumentationVersion = "otlp.version"
+	defaultServiceName     = "otel-collector"
+)
+
+func logDataToLogRecode(ld pdata.Logs) []*logpb.LogData {
```

Comment:
Please, follow the contribution guidelines for new components, specifically the part about breaking the work into multiple PRs.